### PR TITLE
Db functions

### PR DESCRIPTION
**Pull request by hmvassar**
Added a register user function as well as a get user by email function

### CODE REVIEW

File: db/client.js [hunk 1]
```diff
@@ -11,4 +11,4 @@ const client = new Pool({
   port: port,
 });
 
-module.exports = client;
\ No newline at end of file
+module.exports = client;ju
```
Comment (TheBigBooler) [inline db/client.js]: take that "ju" off the end

File: db/users.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const client = require("./client");
+
+//Register function
+async function register({email, name, password }) {
```
Comment (TheBigBooler) [inline db/users.js]: also an address field for the registration

**Review by TheBigBooler — CHANGES_REQUESTED** [review]
fix the client export, add address into registration function

**Review by BullFrog — COMMENTED** [review]
Looks good other than that "ju" error

File: db/users.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    return (error);
+  }
+}
+
+//Get user by email
+async function getUserByEmail(email) {
+try {
+  const { rows: [users] } = await client.query(
+    `
+    SELECT id, email FROM users WHERE id=$1;`,[userId]);
```
Comment (TheBigBooler) [inline db/users.js]: change the userId in client query to email

**Review by TheBigBooler — APPROVED** [review]

**Pull request merged**